src/actions/AuthActions.js

```diff
@@ -61,7 +60,8 @@ export const logoutUser = () => {
         firebase.auth().signOut()
             .then((user) => {
                 console.log("Logged the user out", user)
-                Actions.login();
+
+                //TODO add a trigger here to route back to login screen
```

Comment:
When we go back to working on auth, we'll need to rethink how we do logout and all this.